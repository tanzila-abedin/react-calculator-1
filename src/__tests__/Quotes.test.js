import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const quote = renderer
    .create(<div><h2>Everything you can imagine is real. â€“ Pablo Picasso</h2></div>)
    .toJSON();
  expect(quote).toMatchSnapshot();
});
