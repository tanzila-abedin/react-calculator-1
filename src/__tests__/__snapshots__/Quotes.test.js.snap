// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h2>
    Everything you can imagine is real. â€“ Pablo Picasso
  </h2>
</div>
`;
